concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1440,14 +1479,14 @@ jobs:
     plan:
       - aggregate:
           - get: pipeline-trigger
-            passed: [ 'cf-deploy']
+            passed: ['post-deploy','availability-tests']
```

Comment:
But then if post-deploy runs in parallel with avail tests, it will take the healthcheck app briefly down and thus interfere with avail. tests...